psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="4"
```

Comment:
suggest dropping:

 * `errorLevel`
 * `resolveFromConfigFile`

And then adding:

 * `totallyType="true"`
 * a baseline file